encfs/BlockFileIO.cpp

```diff
@@ -146,7 +146,7 @@ ssize_t BlockFileIO::read(const IORequest &req) const {
 
     // if we're reading a full block, then read directly into the
     // result buffer instead of using a temporary
-    if (partialOffset == 0 && size >= (size_t)_blockSize) {
+    if (partialOffset == 0 && size >= _blockSize) {
```

Comment:
Shouldn't _blockSize be unsigned from the start?

Reply:
It is unsigned now, thanks to this PR.
Here is then remove a wast which is no more needed, as size_t is unsigned.